managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+
+    - **Database Name** (5)
+    - **Host** (13)
+    - **Password** (14)
+    - **Port** (15)
+    - **Username** (32)
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+#### Executing the data import and loading
+
+1. Run the following command to execute the data import and loading:
+
+    ```sh
+    meltano run tap-postgres target-postgres
+    ```
+
+2. When finished, make sure your data is stored in your new database by connecting to it, and performing a query.
+
+    <Message type="note">
+    By default, the `tap-postgres` extractor loads data in the `tap_postgres` schema, rather than the `public` schema. You can find your loaded tables with the following command:
+
+    ```
+    SELECT * from tap-postgres.{table1name} LIMIT 10;
+    ```
+    </Message>
+
+## Importing data from an existing PostgreSQL backup
+
+### Using pg_restore
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- Installed PostgreSQL 16 (or newer) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
+
+- A backup file for your database (named `my-backup` in the following procedure). 
+
+1. In a terminal, access the directory containing the backup file, then run the command below to import data to your Managed Database for PostgreSQL using `pg_restore`. Make sure to replace the placeholders with your Managed Database for PostgreSQL connection parameters:
+
+    ```
+    pg_restore --no-privileges --no-owner --clean --if-exists -U {username} -h {host} --port 5432 -d {databasename} my-backup
+    ```
+
+    <Message type="note">
+    You can find this information on the [overview page](https://console.scaleway.com/rdb/instances) of your database. The password is the [IAM secret key](/identity-and-access-management/iam/concepts/#api-key) corresponding to the [IAM user](/identity-and-access-management/iam/concepts/#user) or [IAM application](/identity-and-access-management/iam/concepts/#application) you are connecting with. 
+    </Message>
+
+2. When finished, make sure your data is stored in your new database by [connecting to it](/managed-databases/postgresql-and-mysql/how-to/connect-database-instance/), and performing a query.
+
+## Import Data from a file (.csv, .txt, etc.)
+
+### Using psql
+
+#### Prerequisites
+
+To complete the following procedure, you must have:
+
+- Installed PostgreSQL 16 (or newer) with [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```

Comment:
We have to adapt a little bit to say that the user must have installed PostgreSQL with pg_dump and pg_restore in the same version as the one of their Managed PostgreSQL